src/components/Widget.tsx

```diff
@@ -154,7 +154,10 @@ export function TestableWidget(props: PropsWithChildren<TestableWidgetProps>) {
         props.jsonRpcUrlMap[supportedChainId as number] = fallbackRpc
       }
     }
-    return props.jsonRpcUrlMap
+    return Object.entries(props.jsonRpcUrlMap).reduce(
```

Comment:
can you make a function for this or name the result as a variable? will make it clearer what's going on

Reply:
I split this into so that Widget.tsx only deals with fallbacks, so that this callback only has a single function.
Converting from `string|string[]` to `string[]` is now done in the child component.